package.json

```diff
@@ -8,19 +8,16 @@
   "bin": {
```

Comment:
This was before I removed json from prettiers list, happy to revert

Reply:
What?